kotlinx-coroutines-core/js/src/flow/internal/SafeCollector.kt

```diff
@@ -17,7 +18,8 @@ internal actual class SafeCollector<T> actual constructor(
     private var lastEmissionContext: CoroutineContext? = null
 
     override suspend fun emit(value: T) {
-        val currentContext = coroutineContext
+        val currentContext = currentContext()
```

Comment:
Extra suspension point! Not a big deal, but still....

Reply:
`currentCoroutineContext` is an inline suspend fun, it does not create suspension point